packages/frontend/templates/build.mjs

```diff
@@ -0,0 +1,26 @@
+import fs from 'fs';
+import * as glob from 'glob';
+import path from 'path';
+
+// purpose: bundle all json files into one json file in onboarding folder
+const __dirname = new URL('.', import.meta.url).pathname;
+
+const content = Object.fromEntries(
+  glob
+    .sync('./*.json', {
+      cwd: path.resolve(__dirname, 'onboarding'),
+    })
+    .map(file => {
+      return [
+        file,
+        JSON.parse(
+          fs.readFileSync(path.resolve(__dirname, 'onboarding', file), 'utf8')
+        ),
+      ];
+    })
+);
+
+fs.writeFileSync(
+  path.resolve(__dirname, 'templates.gen.ts'),
+  `export const onboarding = ${JSON.stringify(content)};`
```

Comment:
```suggestion
  `export const onboarding = ${JSON.stringify(content, null, 2)};`
```

let the generated file readable.

Reply:
Or, better code is importing json directly, not read all json content.

Reply:
done